### ISSUE TITLE
DPI scaling is through the roof

### ISSUE THREAD

**Issue by brittyazel** [flathub/net.scribus.Scribus#1]
![screenshot from 2018-09-14 14-35-41](https://user-images.githubusercontent.com/3747347/45576352-a7049500-b82b-11e8-91ff-d02bf9f798b9.png)

On a new install the scaling seems to be WAY overboard. I'm using a 1080p screen and the window was so big I couldn't even see the window frames. And this is a screenshot when I open up a new document

**Comment by jurf** [comment]
Woah, OK. I saw this bug once before, but we couldn’t figure out what it was. We need to disable this.

Did you overriding the theme in the settings?

**Comment by brittyazel** [comment]
Nope, and in the settings it's the same no matter if I choose adwaita, breeze, windows, whatev. It's all the same

**Comment by jurf** [comment]
This any of [this](https://wiki.archlinux.org/index.php/HiDPI#Qt_5) help? If yeah, please post what, I’ll add it by default.

**Comment by brittyazel** [comment]
from what I tried, none of that did anything. None of it seemed to do anything to any of my QT5 apps either. I dunno. All of my QT apps look fine but this one

**Comment by jurf** [comment]
I have to go now (for the night), but I’d like to get this fixed ASAP. I’d really appreciate it if you’d ask over on `#scribus` on Freenode if you have the time.

**Comment by jurf** [comment]
Did you try this also?

> If you manually set the screen factor, it is important to set `QT_AUTO_SCREEN_SCALE_FACTOR=0` otherwise some applications which explicitly force high DPI enabling get scaled twice.

Looks it this could be the case.

**Comment by jurf** [comment]
Upstream forum post: http://forums.scribus.net/index.php/topic,3063.msg14249.html

**Comment by jurf** [comment]
When, on IRC, playing around with `QT_SCREEN_SCALE_FACTORS` seemed to produce some results, you could try that. You can try setting it to `2`, looks like the default is `3`.

**Comment by jurf** [comment]
And we now have an upstream issue too: https://bugs.scribus.net/view.php?id=15425

**Comment by brittyazel** [comment]
I did they those yes. Unless I'm doing it wrong. How do you set the environment variable at runtime? I tried putting the variable (sans the 'export') in my /etc/environment and I stopped me from being able to boot my laptop, so instead I was just executing the export commands from in a bash prompt. I'm not sure if this is how to do it

**Comment by jurf** [comment]
You can just try `env QT_SCREEN_SCALE_FACTORS=2 flatpak run net.scribus.Scribus`, or `flatpak run --env=QT_SCREEN_SCALE_FACTORS=2 net.scribus.Scribus`.

**Comment by brittyazel** [comment]
Ok that worked! Setting it to QT_SCREEN_SCALE_FACTORS=2 didn't change anything, but changing it to QT_SCREEN_SCALE_FACTORS=1 made it look perfect. So it seems for some reason it's defaulting to 200% scaling, which is wrong

**Comment by nedrichards** [comment]
You should bring this up to the Qt and KDE Runtime people as well if hidpi detection is generally wrong.

**Comment by jurf** [comment]
@nedrichards I don’t think HiDPI detection is generally wrong, this looks like just a Scribus issue.

**Comment by nedrichards** [comment]
@jurf I think we've seen people reporting similar things in VLC fwiw but good news if it's just Scribus.

**Comment by jurf** [comment]
@nedrichards dunno, but this issue was documented in Scribus before the first Flatpak, so it’s probably not Flatpak specific. Though please do post back if you see this elsewhere.

**Comment by jurf** [comment]
New upstream issue: https://bugs.scribus.net/view.php?id=14335

**Comment by eszlari** [comment]
This is works here (200%) with the current version.

**Comment by hfiguiere** [comment]
this was fixed with more recent version of the toolkit
